### PR TITLE
changes in auth reducer

### PR DESCRIPTION
**Pull request by praveen021097**

### CODE REVIEW

File: src/Redux/UserSideBookOneOnOneReducer/Action.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// import { ActionTypes } from "./ActionTypes";
```
Comment (Shivamyadav07) [inline src/Redux/UserSideBookOneOnOneReducer/Action.ts:1]: remove commented coad

File: src/Redux/UserSideBookOneOnOneReducer/Action.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+// export interface isLoginSuccess{
```
Comment (sivagurubilli) [inline src/Redux/UserSideBookOneOnOneReducer/Action.ts:3]: remove commented code

File: src/Pages/BookInterviews/BookOneOnOne.tsx [hunk 3]
```diff
@@ -62,30 +62,29 @@ const BookOneOnOne = () => {
...
                                             </Tr>
                                         </Thead>
                                         <Tbody textAlign={"center"}>
                                             <Tr fontSize={"15px"} alignContent={"center"} bgColor={"rgba(12, 135, 172, 0.318)"}>
-                                                <Td  textAlign={"center"}>Ankush</Td>
-                                                <Td  textAlign={"center"}>interview</Td>
-                                                <Td  textAlign={"center"}>DSA dfdg fgh</Td>
-                                                <Td  textAlign={"center"} color={"blue"} textDecoration={"underline"}><Link to={"#"}>www.googleffhgfhfghghghfgd.com</Link></Td>
-                                                <Td  textAlign={"center"}>9.30 to 10.30</Td>
+                                                <Td  textAlign={"center"}><Text fontSize={"17px"} color={"black"}  >Ankush</Text></Td>
```
Comment (sivagurubilli) [inline src/Pages/BookInterviews/BookOneOnOne.tsx:70]: format this file

Comment (praveen021097) [reply src/Pages/BookInterviews/BookOneOnOne.tsx:70]: formated

**Pull request merged**